test/unit_test/components/gantry_test.dart

```diff
@@ -119,9 +119,9 @@ void main() {
     late Server server;
 
     setUp(() async {
-      final port = 50000 + (StackTrace.current.hashCode % 10000);
+      final port = 50000 + (name.hashCode % 10000);
```

Comment:
seems like a good function to pull out into some kind of a testutils file

Reply:
sgtm

Reply:
https://github.com/viamrobotics/viam-flutter-sdk/pull/41 includes a helper function